src/patternfly/demos/Toolbar/examples/toolbar-complex-example.hbs

```diff
@@ -0,0 +1,59 @@
+{{#> toolbar}}
+  {{#> toolbar-section pf-l-toolbar--modifier="pf-u-justify-content-space-between pf-u-mx-xl pf-u-my-md"}}
+    {{#> toolbar-group}}
+      {{#> toolbar-item pf-l-toolbar__item--modifier="pf-u-mr-2xl"}}
+        {{#> input-group }}
+          {{#> dropdown id="dropdown-example-collapsed" pf-c-dropdown--HasToggleIcon="true"}}
+            All
+          {{/dropdown}}
+          {{#> form-control controlType="input" input="true" type="text" id="textInput10" name="textInput10"  placeholder="Filter..." aria-label="filter input with popup button"}}
+          {{/form-control}}
+          {{#> button btnClass="pf-m-tertiary" btnAttributes='id="textAreaButton2"'}} <i class="fas fa-search aria-hidden="true"></i> {{/button}}
+        {{/input-group}}
+      {{/toolbar-item}}
+      {{#> toolbar-item pf-l-toolbar__item--modifier="pf-u-mr-lg"}}
+        {{#> dropdown id="dropdown-example-collapsed" pf-c-dropdown--HasToggleIcon="true"}}
+          All
+        {{/dropdown}}
+      {{/toolbar-item}}
+      {{#> toolbar-item}}
+        {{#> button btnClass="pf-c-button pf-m-plain" btnAttributes="aria-label=\"Sort A-Z\" "}}
```

Comment:
You have a bunch of escaped double-quotes in the `attribute` list in both demos. You should use a single quote to wrap the `attribute` list instead, and then you won't need to escape the quotes. You can look above on line 11 of this file for an example of that.

You also have an empty space at the end of these lines that looks like it should be removed.

Reply:
thanks for picking up on that